src/main/groovy/edu/oregonstate/mist/locations/mapper/LocationMapper.groovy

```diff
@@ -139,4 +140,34 @@ class LocationMapper  {
         setLinks(resourceObject)
         resourceObject
     }
+
+    /**
+     *  Create a GeoLocation object for Attributes
+     * @param latitude
+     * @param longitude
+     * @return
+     */
+    private GeoLocation createGeoLocation(String latitude, String longitude) {
+        if (convertStringToDouble(latitude) == null || convertStringToDouble(longitude) == null){
```

Comment:
I think you can use: "latitude as Double" and "longitude as Double" to convert the values. If you are checking if something is a double you could use "foo".isDouble() . That would allow you to get rid of an extra method.